src/shared/modules/dbMeta/dbMetaDuck.ts

```diff
@@ -623,6 +658,12 @@ export const serverConfigEpic = (some$: any, store: any) =>
             updateUserCapability(USER_CAPABILITIES.serverConfigReadable, true)
           )
           store.dispatch(updateSettings(settings))
+          if (
+            supportsEditorHistorySetting(store.getState()) &&
+            !settings['browser.retain_editor_history']
+          ) {
+            store.dispatch(clearHistory())
```

Comment:
Would this not always clear history on community version 4.3?

Reply:
ah good catch, looks like you're right, I'll look into fixing this.

Reply:
Any reason we don't use the "shouldRetainEditorHistory" getter?